CHANGES.rst

```diff
@@ -4,6 +4,12 @@ Latest changes
 Release 0.14.0
 --------------
 
+- Improve the load balancing between joblib's workers, which yielded a
+  performance increase for an extensive set of use-cases (including running
+  tasks with varying running time, and combined usage of ``joblib.Memory`` and
+  ``joblib.Parallel``)
```

Comment:
The phrasing is weird. Why use the past tense when the performance is/was better?

May I suggest the following instead:

Improved the load balancing between workers to avoid stragglers caused by an excessively
large batch size when the task duration is varying significantly (because of the
combined use of ``joblib.Parallel`` and ``joblib.Memory`` with a partially warmed cache
for instance).

Reply:
Sure.